### PR TITLE
Changed column resizer ui

### PR DESCRIPTION
**Pull request by Aayushi001**
Added separate event handlers for triggering column resizing and sorting functions.

UI changes:
- Removed the table header line which was previously used for resizing columns. (added screenshot for reference)
- Added border and pseudo-elements to the column headers. 
- CSS changes for proper positioning of icons and pseudo-elements. 

new-col-resizer
old-col-resizer

### CODE REVIEW

File: src/components/VuetableRowHeader.vue [hunk 1]
```diff
@@ -12,24 +12,24 @@
             :class="headerClass('vuetable-th-component', field)"
             :style="{width: field.width}"
             @vuetable:header-event="vuetable.onHeaderEvent"
-            @click="onColumnHeaderClicked(field, $event)"
+            @mousedown="onColumnHeaderClicked(field, $event)"
```
Comment (tarun1793) [inline src/components/VuetableRowHeader.vue]: sorting should only be fired on click and not on mousedown. Please keep the click event for that.

**Comment by tarun1793** [comment]
@Aayushi001 please remove the console log statements, and provide proper description in the PR as to what change(s) have been made. Since this will be a UI change, some screenshots will be of great help.

File: src/components/VuetableRowHeader.vue (hunk 1 above)
Comment (Aayushi001) [reply src/components/VuetableRowHeader.vue]: Done.

File: src/components/VuetableRowHeader.vue [hunk 2]
```diff
@@ -187,8 +190,29 @@ export default {
     },
 
     onColumnHeaderClicked (field, event) {
-      this.vuetable.orderBy(field, event)
+      if(this.vuetable.isDragged){
```
Comment (tarun1793) [inline src/components/VuetableRowHeader.vue:193]: Did it pass the lint test?

Comment (Aayushi001) [reply src/components/VuetableRowHeader.vue:193]: There's no plugin installed for lint checks, so it never actually went through one.
Wrong indentation anywhere?

**Review by tarun1793 — APPROVED** [review]

**Pull request merged**